code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1604,21 +1604,102 @@
 	chem_flags = CHEMICAL_BASIC_ELEMENT
 	taste_description = "metal"
 
-
 /datum/reagent/carpet
 	name = "Carpet"
 	description = "For those that need a more creative way to roll out a red carpet."
 	reagent_state = LIQUID
-	color = "#C8A5DC"
+	color = "#771100" //The color of the average carpet! Carpets are brown, not pink!
 	chem_flags = CHEMICAL_GOAL_BARTENDER_SERVING // this is lame one to put random. at least good as bartender flavor.
```

Comment:
please put chem flag NONE to all subtypes. bartenders will highly get these as their objective goal.

Reply:
Good catch!

changed in 95c35d2